modules/quota-establishment-api.adoc

```diff
@@ -216,16 +219,21 @@ $ curl -k -X GET -H "Authorization: Bearer <token>" -H 'Content-Type: applicatio
 }
 ----
 
-== Rejecting pushes using quota limits
+== Setting _reject_ and _warning_ limits
 
 If an image push exceeds defined quota limitations, a soft or hard check occurs:
 
 * For a soft check, or _warning_, users are notified.
 * For a hard check, or _reject_, the push is terminated.
 
-=== Setting reject and warning limits
```

Comment:
as above, I would prefer not to have sub-procedures for now. We could split up the UI and API documents into smaller modules for each sub-procedure, but for now, i am happy to leave them as one large procedure